Add content to Angular Roadmap - Understanding Basics (#2005)
kamranahmedse/developer-roadmap: content/roadmaps/104-angular/content/102-angular-basics/103-angular-modules.md

> # Angular Modules
> 
> Modules in Angular acts like a container where we can group the components, directives, pipes, and services, which basically related to the application.
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='blue' badgeText='Official Website' href='https://angular.io/guide/architecture-modules'>Introduction to Modules</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.w3schools.com/angular/angular_modules.asp'>Angular Modules</BadgeLink>
> <BadgeLink badgeText='Watch' href='https://www.youtube.com/watch?v=mXtEhgRpRa4'>What is Module?</BadgeLink>

Reviewer: ```suggestion
# Angular Modules

Modules in Angular act like a container where we can group the components, directives, pipes, and services, related to the application.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='Official Website' href='https://angular.io/guide/architecture-modules'>Introduction to Modules</BadgeLink>
```
